demo/Vagrantfile

```diff
@@ -22,15 +24,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.synced_folder ".", "/vagrant", disabled: true
 
-  config.vm.provision "shell", privileged: false, inline: <<SCRIPT
-    ## Uncomment these lines to update flynn before starting
-    # sudo apt-get update -qq
-    # sudo apt-get install -y flynn-host
-    # sudo flynn-host download /etc/flynn/version.json
-
-    sudo start flynn-host
+  config.vm.provider "virtualbox" do |v|
+    v.gui = true
```

Comment:
Oh sweet @lmars. How do I rebuild and test the box image? Looks like you're using packer.